### PR TITLE
OSIDB-2745: Remove flaw comment type field

### PR DESCRIPTION
**Pull request by MrMarble**
`FlawComment.type` has a single possible value and therefore it is no needed at all. This task is to remove it together with `FlawComment.FlawCommentType`.

This is part of the `4.0` release

Closes OSIDB-2745

### CODE REVIEW

**Review by osoukup — APPROVED** [review]
LGTM

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove "resolution" field from Flaw (OSIDB-2737)
 - Remove several cvss fields from Flaw (OSIDB-2749)
 - Remove several cvss fields from Affect (OSIDB-2749)
+- Remove "statypete" field from Flaw Comment (OSIDB-2745)
```
Comment (jobselko) [inline docs/CHANGELOG.md:45]: typo `statypete`

Comment (jobselko) [reply docs/CHANGELOG.md:45]: also use the model name, i.e. `FlawComment`

Comment (MrMarble) [reply docs/CHANGELOG.md]: Good catch! changed!

**Review by jobselko — APPROVED** [review]

**Pull request merged**